integration_tests/mocks/rollback_event_repo.py

```diff
@@ -0,0 +1,33 @@
+from typing import Optional
+from pytest import MonkeyPatch
+
+from sqlalchemy.exc import NoResultFound
+
+from app.model.event import EventCreateDTO, EventReadDTO
+
+
+def mock_rollback_event_repo(event_repo, monkeypatch: MonkeyPatch, mocker):
+    # actual_update = event_repo.update
+    actual_create = event_repo.create
+    # actual_delete = event_repo.delete
+
+    # def mock_update_document(db, import_id: str, data: EventReadDTO) -> EventReadDTO:
+    #     actual_update(db, import_id, data)
+    #     raise NoResultFound()
+
+    def mock_create_document(db, data: EventCreateDTO) -> Optional[EventReadDTO]:
+        actual_create(db, data)
+        raise NoResultFound()
+
+    # def mock_delete_document(db, import_id: str) -> bool:
```

Comment:
Nice pre-emptive coding :wink:

Reply:
Whoops - great spot. Have removed all the commented out code in this file.